test/Microsoft.AspNetCore.Hosting.Tests/WebHostBuilderTests.cs

```diff
@@ -1068,6 +1095,24 @@ public void Configure(IWebHostBuilder builder)
             }
         }
 
+        public class StartupWithResolvedDisposableThatThrows
+        {
+            public StartupWithResolvedDisposableThatThrows(DisposableService service)
+            {
+
+            }
+
+            public void ConfigureServices(IServiceCollection services)
+            {
+                throw new InvalidOperationException();
+            }
+
+            public void Configure(IApplicationBuilder app)
```

Comment:
nit: continue the trend where empty methods are shortened as `public void Foo(...) {}`?

Reply:
I don't think that's a consistent trend.